packages/webview_flutter/webview_flutter_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.5.0
+
+* Add an option to set the background color of the webview.
```

Comment:
nit: prefer using present tense when describing changes in the CHANGELOG per new [style guide](https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style):
```suggestion
* Adds an option to set the background color of the webview.
```